cmd/expiration-mailer/main.go

```diff
@@ -447,15 +447,16 @@ func initStats(stats prometheus.Registerer) mailerStats {
 	sendLatency := prometheus.NewHistogram(
 		prometheus.HistogramOpts{
 			Name:    "send_latency",
-			Help:    "Time the mailer takes sending messages",
+			Help:    "Time the mailer takes sending messages in seconds",
 			Buckets: metrics.InternetFacingBuckets,
 		})
 	stats.MustRegister(sendLatency)
 
 	processingLatency := prometheus.NewHistogram(
 		prometheus.HistogramOpts{
-			Name: "processing_latency",
-			Help: "Time the mailer takes processing certificates",
+			Name:    "processing_latency",
+			Help:    "Time the mailer takes processing certificates in seconds",
+			Buckets: []float64{1, 15, 30, 60, 75, 90, 120},
```

Comment:
I like that better. If you find you need it more granular in a place it can be added later. Nice work!